src/codegeneration/TempVarTransformer.js

```diff
@@ -110,13 +113,15 @@ class VarScope {
 export class TempVarTransformer extends ParseTreeTransformer {
   /**
    * @param {UniqueIdentifierGenerator} identifierGenerator
+   * @param {Options} options
    */
-  constructor(identifierGenerator) {
+  constructor(identifierGenerator, options) {
```

Comment:
Making ParseTreeTransformer take `identifierGenerator, reporter, options` caused a lot of churn. Gave up on that. I'll try to TempVarTransformer now.